packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -749,6 +749,100 @@ describe(CreateRepositoryAttributeUseCase.name, () => {
         const attribute = result.value;
         expect(attribute.isDefault).toBeUndefined();
     });
+
+    describe("validation errors for the attribute content", () => {
```

Comment:
Please write "Attribute" capitalized.

Reply:
I'd rather leave the test namings to a different discussion/PR - capitalization is infrequent there :)